### DISCUSSION TITLE
[Mobile] Hide Cursor at readonly mode

### DISCUSSION THREAD

**Discussion by MeguRobert** [singerdmx/flutter-quill#751] [Mobile]
My issue is about [Mobile] 

There is no option to hide the Editor's cursor. At least at the readonly mode I think it would be useful.

**Comment by agil3st** [comment]
This one works for me. #139 by @WingCH
Just change `focusNode: FocusNode()` if you need.

**Comment by NeroSong** [comment]
Maybe you can try to wrap it with an `AbsorbPointer`, it works for me.
